bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -1026,10 +1026,11 @@ public String toString() {
     }
 
     @Override
-    public void add(Item element) {
+    public Item add(Item element) {
         if (itemRegistry != null) {
-            itemRegistry.add(element);
+            return itemRegistry.add(element);
         }
+        return element;
```

Comment:
"if no ManagedProvider is available" -> this is already thrown by the itemRegistry.add() call.
Now we throw an IllegalStateException if the itemRegistry is not available - this is a changed behavior and not mentioned in the JavaDoc, we should not do this.